scripts/publish-npm.js

```diff
@@ -0,0 +1,121 @@
+const { execSync } = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+const readline = require('readline');
+
+const rootPath = path.resolve('__dirname', '..');
+
+function requireSegmentApiKey() {
+  const MONGOSH_SEGMENT_API_KEY = process.env.MONGOSH_SEGMENT_API_KEY || '';
+
+  if (!MONGOSH_SEGMENT_API_KEY) {
+    throw new Error('MONGOSH_SEGMENT_API_KEY environment variable must be set');
+  }
+
+  if (MONGOSH_SEGMENT_API_KEY.length !== 32) {
+    throw new Error('MONGOSH_SEGMENT_API_KEY does not seem to have the proper format');
```

Comment:
I feel like this could also be an `assert`.

Reply:
makes sense